docs/repro.md

```diff
@@ -0,0 +1,21 @@
+# Bug Report Reproduction Guide
+
+When customers plan to report an issue with the ASP.NET Core framework, we will most likely ask them to provide a so called `minimal repro project`.
+This document describes what a minimal repro project is, and why it's important to us.
+
+## What is a minimal repro project?
+A repro, or a reproduction is a project, which can be used to reproduce the reported behavior with minimal effort from a product team, which has the minimum code required to demonstrate the concerning behavior.
+
+Here are the steps for creating such a project:
+- Create a new project, based on one of the ASP.NET Core project templates.
+- Add the minimum amount of code necessary to reproduce the behavior you try to report on this newly created project
+- Make sure you don't add any dependencies, which are irrelevant to the behavior.
```

Comment:
```suggestion
- Make sure you don't add any dependencies that are irrelevant to the behavior.
```

The current phrasing makes it sound like you're asking them to never add any dependencies at all

Reply:
Thanks @wtgodbe 👍